main.gms

```diff
@@ -264,7 +264,6 @@ $setGlobal downscaleTemperature  off  !! def = off
 *'---------------------    20_growth    ------------------------------------------
 *'
 *' * (exogenous): exogenous growth
-*' * (endogenous): endogenous growth !!Warning: still experimental stuff!!
```

Comment:
This change has no relation with the removal of `pm_fe_demand` as far as I can see. Is this intentional?

Reply:
yes, just cleaning. this realization doe not exist anymore